packages/core/build/tokens/tokens.ts

```diff
@@ -0,0 +1,542 @@
+      300: token('0 1px 3px 0 rgba(27, 43, 50, 0.2)'),
+    },
+    opacity: {
+      100: token('0.2'),
+      200: token('0.4'),
+      300: token('0.8'),
+    },
+    interaction: {
+      background: {
+        value: token(color.gray[0]),
+        hover: token(color.gray[100]),
+        active: token(color.gray[200]),
+        selected: token(color.blue[50]),
+        disabled: token(color.gray[0]),
+        highlight: token(color.blue[700]),
+      },
+      color: {
+        value: token(color.construction[700]),
+        hover: token(color.construction[1000]),
+        active: token(color.construction[1000]),
+        selected: token(color.construction[700]),
+        disabled: token(color.construction[200]),
+      },
+    },
+    app: {
+      background: token(color.gray[50]),
+    },
+    overlay: {
+      background: token(color.gray[0]),
+      backdropBackground: token(color.gray[900]),
+    },
+    container: {
+      background: token(color.gray[0]),
+      borderColor: token(color.construction[200]),
+    },
+  },
+  status: {
+    info: {
+      value: token(color.blue[700]),
+      tint: token(color.blue[600]),
+      shade: token(color.blue[800]),
+    },
+    success: {
+      value: token(color.green[700]),
+      tint: token(color.green[600]),
+      shade: token(color.green[800]),
+    },
+    warning: {
+      value: token(color.ochre[800]),
+      tint: token(color.ochre[700]),
+      shade: token(color.ochre[900]),
+    },
+    danger: {
+      value: token(color.red[700]),
+      tint: token(color.red[600]),
+      shade: token(color.red[800]),
+    },
+    neutral: {
+      value: token(color.construction[600]),
+      tint: token(color.construction[400]),
+      shade: token(color.construction[700]),
+    },
+    disabled: {
+      value: token(color.construction[300]),
+      tint: token(color.construction[200]),
+      shade: token(color.construction[400]),
+    },
+    alt: {
+      value: token(color.violet[700]),
+      tint: token(color.violet[600]),
+      shade: token(color.violet[900]),
+    },
+    secondary: {
+      info: {
+        value: token(color.blue[800]),
+        tint: token(color.blue[50]),
+        shade: token(color.blue[100]),
+      },
+      success: {
+        value: token(color.green[800]),
+        tint: token(color.green[50]),
+        shade: token(color.green[100]),
+      },
+      warning: {
+        value: token(color.ochre[800]),
+        tint: token(color.ochre[50]),
+        shade: token(color.ochre[100]),
+      },
+      danger: {
+        value: token(color.red[800]),
+        tint: token(color.red[50]),
+        shade: token(color.red[100]),
+      },
+      neutral: {
+        value: token(color.construction[600]),
+        tint: token(color.construction[50]),
+        shade: token(color.construction[100]),
+      },
+      alt: {
+        value: token(color.violet[800]),
+        tint: token(color.violet[50]),
+        shade: token(color.violet[100]),
```

Comment:
The `token()` function allows the build to create references to other tokens so it can generate css vars that use other css vars. Example a success alias color uses one of the global green colors.